src/components/CheckEditor/CheckEditor.tsx

```diff
@@ -151,11 +151,20 @@ export const CheckEditor = ({ check, onReturn }: Props) => {
             frequency={check?.frequency ?? fallbackCheck.frequency}
             probes={check?.probes ?? fallbackCheck.probes}
           />
-          <HorizontalCheckboxField
-            id="useFullMetrics"
-            name="useFullMetrics"
-            label="Publish full set of metrics"
-            description={'Metrics are reduced by default'}
+          <Controller
```

Comment:
At second glance, I'm not sure why/if this is necessary. The `HorizontalCheckboxField` component looks like it _should_ be registering with react-hook-form. I think it's OK because `HorizontalCheckboxField` will only try and register if it has a `name` prop (which is doesn't here), but now I'm concerned that maybe the checkbox field isn't working in the other places it's getting used